src/outputs/parthenon_hdf5.cpp

```diff
@@ -428,13 +430,12 @@ void PHDF5Output::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm) {
       tmpData[i] = -1.25;
 
     // Write mesh coordinates to file
-    hsize_t local_start[5], global_count[5], local_count[5];
+    hsize_t local_start[7], global_count[7], local_count[7];
```

Comment:
@pgrete sorry for delay - my work on parthenon is sporadic these days.
**Why is this change required and how is it connected to block variables?**
On a downstream code I'm working on I need to store a per-block basis.  Parthenon already allows this to be done via `Metadata::None` flag.  However the HDF output does not allow me to write this variable to the `phdf` file so that I can pick it up post run.  This change is to allow me to output the per-block variable as per-block instead of per-cell which would expand it by 40k times.

**How does this affect dimension of the other variables?**
It doesn't affect any of the other variables at all.  It just so happens that to declare it I have to declare it as the full 6-dimensional array.

** Also, does this have any implications on the xdmf output and/or our python scripts to read data?**
It gets put out in the `XDMF` as a `Grid` type variable (this PR).  In Python it will be available as usual using `h5py`

Please let me know if I've answered your questions.